lib/input_validator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "board"
+
+class InputValidator
+    def initialize
+        @invalid_moves = []
+    end
+
+    def valid_character?(move)
+        regex = /\D/ 
```

Comment:
What's a more descriptive name for this?

Reply:
Maybe valid_coordinate, valid_position, valid_selection?